src/petals/server/throughput.py

```diff
@@ -10,9 +10,8 @@
 import torch
 from hivemind.utils.logging import get_logger, use_hivemind_log_handler
 
-from petals.bloom.block import BloomBlock
-from petals.bloom.model import BloomConfig
-from petals.bloom.ops import build_alibi_tensor
+from petals.bloom.block import WrappedBloomBlock
+from petals.bloom.modeling_utils import BloomConfig
```

Comment:
```suggestion
from transformers import BloomConfig
```

Reply:
Better to make it clear that we do not override/reimplement the config, having it imported in `petals.bloom.modeling_utils` is more of a coincidence